chapters/ch35.md

```diff
@@ -1,43 +1,35 @@
 
 > For the subset of data with rapidly spoken speech:
 
---> _replace THIS LINE by your translation for the above line_
-
+Đối với tập dữ liệu con với lời nói nhanh:
 
 > 1. You can still obtain transcripts from humans that are higher quality than your algorithm’s output.
 
---> _replace THIS LINE by your translation for the above line_
-
+1. Bạn vẫn có thể nhận được bảng điểm từ con người với chất lượng cao hơn so với đầu ra thuật toán của bạn.
 
 > 2. You can draw on human intuition to understand why they correctly heard a rapidly spoken utterance when your system didn’t.
 
---> _replace THIS LINE by your translation for the above line_
-
+1. Bạn có thể dựa vào trực giác để hiểu lý do tại sao họ nghe chính xác một phát ngôn nhanh khi hệ thống của bạn không thành công.
 
 > 3. You can use human-level performance on rapidly spoken speech as a desired performance target.
 
---> _replace THIS LINE by your translation for the above line_
-
+1. Bạn có thể dùng chất lượng ở cấp độ con người trên lời nói nhanh như một mục tiêu với chất lượng mong muốn.
 
 > More generally, so long as there are dev set examples where humans are right and your algorithm is wrong, then many of the techniques described earlier will apply. This is true even if, averaged over the entire dev/test set, your performance is already surpassing human-level performance.
 
---> _replace THIS LINE by your translation for the above line_
-
+Tổng quát hơn, miễn là có các ví dụ về tập phát triển nơi con người đúng và thuật toán của bạn sai, thì nhiều kỹ thuật được mô tả trước đó sẽ được áp dụng. Điều này đúng ngay cả khi, tính trung bình trên toàn bộ tập phát triển/kiểm tra, chất lượng của bạn đã vượt qua chất lượng ở cấp độ con người.
 
 > There are many important machine learning applications where machines surpass human level performance. For example, machines are better at predicting movie ratings, how long it takes for a delivery car to drive somewhere, or whether to approve loan applications. Only a subset of techniques apply once humans have a hard time identifying examples that the algorithm is clearly getting wrong. Consequently, progress is usually slower on problems where machines already surpass human-level performance, while progress is faster when machines are still trying to catch up to humans.
 
---> _replace THIS LINE by your translation for the above line_
-
+Có nhiều ứng dụng học máy quan trọng trong đó máy móc vượt qua chất lượng của con người. Ví dụ, máy móc tốt hơn trong việc dự đoán xếp hạng phim, sẽ mất bao lâu để một chiếc xe giao hàng lái xe đi đâu đó, hoặc có chấp nhận đơn xin vay tiền hay không. Chỉ một tập hợp các kỹ thuật được áp dụng một khi con người gặp khó khăn trong việc xác định các ví dụ rằng thuật toán rõ ràng đang bị sai. Do đó, tiến độ thường chậm hơn đối với các vấn đề trong đó máy móc đã vượt qua chất lượng ở cấp độ con người, trong khi tiến độ nhanh hơn khi máy móc vẫn đang cố gắng bắt kịp con người.
```

Comment:
- "Chỉ một tập ~hợp~ _nhỏ_ (vì bản gốc là subset) các kỹ thuật ~được~ là áp dụng _được_ ~một~ khi mà con người _còn_ gặp khó khăn trong việc xác định các ~ví dụ~ mẫu ~rằng~ _nào mà_ thuật toán _còn_ rõ ràng đang ~bị~ _làm_ sai."

Reply:
@lkhphuc "subset" là "tập con" nhé.